documentation/api/query/v4/events.markdown

```diff
@@ -187,19 +187,25 @@ For all events in the report with hash
 To retrieve all of the events within a given time period:
 
     ["and", ["<", "timestamp", "2011-01-01T12:01:00-03:00"],
-            [">", "timestamp", "2011-01-01T12:00:00-03:00"]]
+      [">", "timestamp", "2011-01-01T12:00:00-03:00"]]
```

Comment:
yes. It makes it line up with "extract" below. Indenting on the array makes it look weird when the operators aren't all "and".